core-plugins/docs/Email-postaction.md

```diff
@@ -0,0 +1,57 @@
+# Email Post Action
+
+
+Description
+-----------
+Sends an email at the end of a pipeline run.
+
+
+Use Case
+--------
+This action can be used when you want to send an email at the end of a pipeline run.
+For example, you may want to configure a pipeline so that an email is sent whenever
+the run failed for any reason.
+
+
+Properties
+----------
+**recipientEmailAddress:** The address to send the email to.
+
+**senderEmailAddress:** The address to send the email from.
+
+**message:** The message of the email.
+
+**subject:** The subject of the email.
+
+**protocol:** The email protocol to use. smtp, smtps, and tls are supported. Defaults to SMTP.
+
+**username:** The username to use for authentication if the protocol requires it.
+
+**password:** The password to use for authentication if the protocol requires it.
+
+**host:** The SMTP host to use. Defaults to localhost.
+
+**port:** The SMTP port to use. Defaults to 25.
+
+**includeWorkflowToken:** Whether to include the contents of the workflow token in the email message. Defaults to false.
+
+Example
+-------
+This example sends an email from 'team-ops@company.com' to 'team-alerts@company.com' whenever a run fails
```

Comment:
there's nothing in the config to make it execute only on run failure. change the description?